crates/git/Cargo.toml

```diff
@@ -30,6 +30,7 @@ text.workspace = true
 time.workspace = true
 url.workspace = true
 util.workspace = true
+regex.workspace = true
```

Comment:
Let's put this dep on line 25 to keep the list of dependencies sorted.

Reply:
done 🙂